packages/account/src/Sections/Assessment/FinancialAssessment/financial-assessment.jsx

```diff
@@ -297,13 +301,15 @@ const FinancialAssessment = ({
 
     const getScrollOffset = () => {
         if (isMobile()) return is_appstore ? '160px' : '200px';
+        if (isMobile()) return is_mf && is_financial_information_incomplete && !is_submit_success ? '40px' : '80px';
+        if (isDesktop()) return is_mf && is_financial_information_incomplete && !is_submit_success ? '140px' : '80px';
         return '80px';
```

Comment:
When can both `isMobile` and `isDesktop` be false? If they can't, this return will never be reached